statistics/table.go

```diff
@@ -63,6 +60,20 @@ const (
 	PseudoRowCount = 10000
 )
 
+var (
+	// Below functions are used to solve cycle import problem.
+	// TODO: resolve the cycle import problem in a more graceful way.
```

Comment:
This cycle import problem will be naturally resolved as we move more estimation functions from `stats` to `cardinality` package.